lib/feedback.js

```diff
@@ -108,6 +108,18 @@ function feedback(system) {
 		self.system.emit('feedback_save')
 	})
 
+	system.on('feedback_check_all', function () {
```

Comment:
This isnt rechecking any feedbacks used in triggers, but the name suggests it should.
Perhaps call it `feedback_check_all_banks`?